src/main/java/com/sweetwith/dailynote/domain/user/User.java

```diff
@@ -10,23 +10,21 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 
+@NoArgsConstructor
 @Getter
 @Entity
 public class User extends BaseTimeEntity {
+
     @javax.persistence.Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
+    private Long Id;
```

Comment:
How about specify column name? If you don't specify column name, then column would be generated as you declared.
https://www.icatpark.com/entry/JPA-%EA%B8%B0%EB%B3%B8-Annotation-%EC%A0%95%EB%A6%AC
https://wedul.site/507